frontend/src/components/OperationDataTableContainer/OperationDataTableContainer.tsx

```diff
@@ -0,0 +1,68 @@
+import { List } from 'immutable';
+import * as React from 'react';
+import { OperationDataMap } from '../../types/operations';
+import { ColumnList } from '../../types/sources';
+import { fetchOperationData } from '../../pages/QueryData/actions';
+import { OperationDataTable } from '../OperationDataTable';
+
+interface OperationDataTableContainerProps {
+  list?: List<OperationDataMap>;
+  columns?: ColumnList;
+  id?: string;
+  limit: number;
+  offset: number;
+  hidePaginate?: boolean | undefined;
+  fetchData?: typeof fetchOperationData;
+}
+
+const getColumns = (item?: OperationDataMap): string[] => {
+  if (item) {
+    const columns: string[] = [];
+    item.mapKeys((key: string) => columns.push(key));
+
+    return columns;
+  }
+
+  return [];
+};
+
+export const OperationDataTableContainer: React.SFC<OperationDataTableContainerProps> = (props) => {
```

Comment:
I think I've mentioned before that SFC is deprecated ... it may exist in old code, but it should not be used in new code.